README.md

```diff
@@ -1,2 +1,86 @@
-# pupmod-simp-cron
-A SIMP Puppet module for managing cron
+[![License](http://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html) [![Build Status](https://travis-ci.org/simp/pupmod-simp-cron.svg)](https://travis-ci.org/simp/pupmod-simp-cron) [![SIMP compatibility](https://img.shields.io/badge/SIMP%20compatibility-6.0.*-orange.svg)](https://img.shields.io/badge/SIMP%20compatibility-6.0.*-orange.svg)
+
+
+#### Table of Contents
+
+1. [Description](#description)
+2. [Setup - The basics of getting started with cron](#setup)
+    * [What cron affects](#what-cron-affects)
+3. [Usage - Configuration options and additional functionality](#usage)
+4. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+5. [Limitations - OS compatibility, etc.](#limitations)
+6. [Development - Guide for contributing to the module](#development)
+    * [Acceptance Tests - Beaker env variables](#acceptance-tests)
+
+
+## Description
+
+This module manages the cron packges, service, and /etc/cron.allow.
+
+
+### This is a SIMP module
+This module is a component of the [System Integrity Management Platform](https://github.com/NationalSecurityAgency/SIMP), a compliance-management framework built on Puppet.
+
+If you find any issues, they can be submitted to our [JIRA](https://simp-project.atlassian.net/).
+
+This module is optimally designed for use within a larger SIMP ecosystem, but it can be used independently:
+
+ * When included within the SIMP ecosystem, security compliance settings will be managed from the Puppet server.
+ * If used independently, all SIMP-managed security subsystems are disabled by default and must be explicitly opted into by administrators.  Please review the `$client_nets`, `$enable_*` and `$use_*` parameters in `manifests/init.pp` for details.
+
+
+## Setup
+
+
+### What cron affects
+
+  * crond service
+  * `/etc/cron.deny`
+  * `/etc/cron.allow`
+  * tmpwatch package (if EL6)
+
+## Usage
+
+To use this module, just call the class. This example adds it to a class list in hiera:
+
+```yaml
+---
+classes:
+  - cron
+```
+
+Users can also be added to `/etc/cron.allow` with the `cron::user` defined type, or
+the `cron::users` array in hiera. The following example adds a few users to `/etc/cron.allow`:
+
+```yaml
+cron::users:
+  - foo
+  - bar
+```
+
+
+## Reference
+
+Please refer to the inline documentation within each source file, or to the module's generated YARD documentation for reference material.
+
+
+## Limitations
+
+SIMP Puppet modules are generally intended for use on Red Hat Enterprise Linux and compatible distributions, such as CentOS. Please see the [`metadata.json` file](./metadata.json) for the most up-to-date list of supported operating systems, Puppet versions, and module dependencies.
+
+
+## Development
+
+Please read our [Contribution Guide] (http://simp-doc.readthedocs.io/en/stable/contributors_guide/index.html).
+
+
+### Acceptance tests
```

Comment:
This module has not acceptance tests.  Perhaps this section should be removed or replace with spec test instructions.